packages/api/lib/publishSnsMessageUtils.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { publishSnsMessage } = require('@cumulus/aws-client/SNS');
+const { envUtils } = require('@cumulus/common');
+
+const Logger = require('@cumulus/logger');
+
+const logger = new Logger({ sender: '@cumulus/publishSnsMessageUtils' });
+
+const publishCollectionSnsMessage = async (record, event) => {
+  let messageToPublish;
+  const topicArn = envUtils.getRequiredEnvVar('collection_sns_topic_arn', process.env);
+
+  if (event === 'Delete') {
+    const deleteRecord = { name: record.name, version: record.version };
+    messageToPublish = { event, record: deleteRecord, deletedAt: Date.now() };
+  } else if (event === 'Create' || event === 'Update') {
+    messageToPublish = { event, record };
+  }
```

Comment:
This is a nitpick, but I slightly prefer this implementation of determining what collection SNS message to send:

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/lambdas/publish-collections.js#L11-L20

It provides a default case if some how the event type didn't match any expected value, but I also just think it's a bit easier to read. I would vote to copy that switch style logic into a function named `createCollectionSnsMessage` or somethign and use it in this function.